apps/profile/schema.py

```diff
@@ -0,0 +1,28 @@
+import graphene
+from graphene_django import DjangoObjectType
+
+from .models import ClientProfile, StylistProfile
+
+
+class ClientProfileModel(DjangoObjectType):
+    class Meta:
+        model = ClientProfile
+
+
+class StylistProfileModel(DjangoObjectType):
+    class Meta:
+        model = StylistProfile
+
+
+class Query(graphene.ObjectType):
+    client_profiles = graphene.List(ClientProfileModel)
+    stylist_profiles = graphene.List(StylistProfileModel)
+
+    def resolve_stylist_profiles(self, info, **kwargs):
+        return StylistProfile.objects.all()
```

Comment:
Can we have an option of returning only one profile? Also for the client

Reply:
@KicaRonaldOkello, i have implemented this